docs/source/getting_started.rst

```diff
@@ -4,9 +4,9 @@ Getting Started
 Installation
 ------------
 
-There are four main ways to install |app|. You can install |app|
-using the Google Play Store version, the F-Droid version, the Amazon app
-store and directly from GitHub releases.
+There are four main ways to install |app| on an Android phone. You can install |app|
+using the Google Play Store, F-Droid, the Amazon App Store and directly from our
+GitHub source code repository.
```

Comment:
Is this clear enough that we mean [releases page](https://github.com/AndBible/and-bible/releases) that has apk files to be downloaded, rather than source code?